fe-nc-news-project/src/utils.js

```diff
@@ -23,12 +22,32 @@ export function fetchArticlesById(article_id) {
         console.log(err, "request error");
     })
 }
-
 export function fetchCommentsByArticleId(article_id) {
     return api.get(`/articles/${article_id}/comments`)
     .then((result) => {
         return result
     }).catch((err) => {
         console.log(err, "request error");
```

Comment:
We don't need .catch in our api file. We want to add a .catch when we invoke the util function in the components.